components/layermanager/layermanager.service.js

```diff
@@ -730,6 +733,7 @@ export default function (
       cur_res = HsMapService.map.getView().getResolution();
     }
     me.currentResolution = cur_res;
+    console.log(cur_res, lyr.getMaxResolution());
```

Comment:
Probably a leftover?

Reply:
O right, thanks. Will resolve with backporting of https://github.com/hslayers/hslayers-ng/pull/1206